Translate README (#1)
line/line-api-use-case-reservation-hairsalon: README.md

# Overview

> This is the demo app source code for the [Hair Salon Reservation](https://lineapiusecase.com/en/usecase/reservation.html) provided on the [LINE API Use Case site](https://lineapiusecase.com/en/top.html). By referring to the steps described in this article, you can develop a hair salon reservation app using the LINE API. The hair salon reservation app is launched on the LIFF browser of the LINE app, and you can make a reservation at the hair salon. To prevent users from forgetting to visit the store, you can implement a reminder function via LINE message by default.

Reviewer: [nitpicking]
It sounds a bit like LIFF browser is a feature within the LINE app, so how about:
```suggestion
This is the demo app source code for the [Hair Salon Reservation](https://lineapiusecase.com/en/usecase/reservation.html) provided on the [LINE API Use Case site](https://lineapiusecase.com/en/top.html). By referring to the steps described in this article, you can develop a hair salon reservation app using the LINE API. The hair salon reservation app is launched on the LIFF browser in the LINE app, and you can make a reservation at the hair salon. To prevent users from forgetting to visit the store, you can implement a reminder function via a LINE message by default.
```

The source code environment introduced on this page uses AWS.

*Documentation and other content are only available in Japanese.

# Libraries

## Node.js

Install Node.js, which will be used for the front-end development, in your local development environment.

*We recommend installing the latest Long-Term Support (LTS) version (v10.13 or later).

> - [Download Node.js](https://nodejs.org/ja/download/)

Reviewer: There is actually an English version of the page:
```suggestion
- [Download Node.js](https://nodejs.org/en/download/)
```

Author: I'm sorry, I missed this link. Thanks!

## Python

Install Python version 3.8 or later if it isn't already installed.

You can check if it's installed by entering this command in Command Prompt or Terminal:

```
python --version
```

If Python is installed, the version will be returned. For example:

```
Python 3.8.3
```

> Install Python (3.8 or higher), used for backend development, in your local development environment if it's not already installed.

Reviewer: In the [previous sentence](https://github.com/line/line-api-use-case-reservation-hairsalon/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R13), `front-end` is used, and here `backend` without a hyphen is used. 
I think both are completely fine, but could you make the syntax consistence?

Author: Sure can

Author: ```suggestion
Install Python (3.8 or higher), used for back-end development, in your local development environment if it's not already installed.
```

Python installation reference site:

Windows: https://docs.python.org/3/using/windows.html
Mac: https://docs.python.org/3/using/mac.html

## AWS SAM

The AWS Serverless Application Model (AWS SAM) is used to deploy this app.

See the [official AWS documentation](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) to register and set up an AWS account, and install the AWS SAM CLI and Docker.

*Recommended version of SAM CLI is 1.15.0 or later.
*Docker installation is also required, with or without local testing.

### Reference points in the official documentation

Complete these items in the official documentation and proceed to the next step. If you have already installed it, you can skip this section.

*This document was created in December 2020 and may be inconsistent with the latest official documentation.

1. Install AWS SAM CLI
1. Set up AWS authentication credentials
1. (Optional) Tutorial: Deploying the Hello World app

# Getting started / Tutorial

> In this section, you'll learn how to create a LINE channel, build the backend and front-end, submit test data, and operation verification, required for app development.

Author: ```suggestion
In this section, you'll learn how to create a LINE channel, build the back-end and front-end, submit test data, and operation verification, required for app development.
```

> In this section, you'll learn how to create a LINE channel, build the backend and front-end, submit test data, and operation verification, required for app development.

Reviewer: ```suggestion
In this section, you'll learn how to create a LINE channel, build the back-end and front-end, submit test data, and operation verification, required for app development.
```

Author: Done

See these steps to build the production environment (AWS) and the local environment:

- **[Create a LINE channel](./docs/liff-channel-create.md)**

> - **[Build the backend](./docs/back-end-construction.md)**

Author: ```suggestion
- **[Build the back-end](./docs/back-end-construction.md)**
```

> - **[Build the backend](./docs/back-end-construction.md)**

Reviewer: ```suggestion
- **[Build the back-end](./docs/back-end-construction.md)**
```

Author: Done
